docs/pages/home/home.yaml

```diff
@@ -332,6 +332,92 @@ paths:
       security:
         - VesselAPIToken: []
     parameters: []
+  /api/passthrough:
+    post:
+      summary: Passthrough Request
+      operationId: passthrough
+      parameters:
+      - name: x-vessel-access-token
+        in: header
+        description: Access Token
+        required: true
+        schema:
+          type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: responseBody
+                description: ''
+                type: object
+                properties:
+                  headers:
+                    type: object
+                  path:
+                    type: string
+                    description: The full URL that the request was sent to.
```

Comment:
this should be the path - you can steal the wording and examples here https://docs.vessel.dev/pages/crm/api-reference/authenticated-passthrough

Reply:
We should return the full url in the response for easier debugging?